xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -549,14 +549,14 @@ void CGUIWindowMusicNav::GetContextButtons(int itemNumber, CContextButtons &butt
         // 4. specific per album
         buttons.Add(CONTEXT_BUTTON_SET_CONTENT, 20195);
       }
-      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtist().size() > 0)
+      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtistString().size() > 0)
```

Comment:
For music video I take random mp4 files and create according nfo by hand containing : 

```    <musicvideo>
        <title>Monster Trucj</title>
        <artist>Bestartistintheworld / aera</artist>
        <album>Me</album>
        <genre>Pop</genre>
        <runtime>3:20</runtime>
        <plot>Scantly clad women hoing about</plot>
        <year>2000</year>
        <director>and I</director>
        <studio>Ego prod.</studio>
    </musicvideo>
```
This usually covers most for Yatse needs, don't know if there's other used field for music videos